datalad/cmdline/helpers.py

```diff
@@ -132,6 +133,55 @@ def __call__(self, parser, namespace, values, option_string=None):
 #        i = 1
 
 
+def parser_add_version_opt(parser, mod_name, include_name=False, delay=False):
+    """Setup --version option
+
+    Parameters
+    ----------
+    parser:
+    mod_name: str, optional
+    include_name: bool, optional
```

Comment:
I don't feel strongly about it, but my vote would be to just stick with the previous output and always include the name.

Reply:
I am also luke warm about this change -- benefit is minimal and it might break someone's (unlikely but possible to exist) script.  May be @mih has an opinion on this minute aspect?